README.md

```diff
@@ -1,3 +1,77 @@
 # Conduit Connector Plugin SDK
 
-Readme coming soon. :construction:
+[![License](https://img.shields.io/badge/license-Apache%202-blue)](https://github.com/ConduitIO/connector-plugin-sdk/blob/main/LICENSE.md)
+[![Build](https://github.com/ConduitIO/connector-plugin-sdk/actions/workflows/build.yml/badge.svg)](https://github.com/ConduitIO/connector-plugin-sdk/actions/workflows/build.yml)
+[![Go Report Card](https://goreportcard.com/badge/github.com/conduitio/connector-plugin-sdk)](https://goreportcard.com/report/github.com/conduitio/connector-plugin-sdk)
+[![Go Reference](https://pkg.go.dev/badge/github.com/conduitio/connector-plugin-sdk.svg)](https://pkg.go.dev/github.com/conduitio/connector-plugin-sdk)
+
+This repository contains the Go software development kit for implementing a connector plugin for
+[Conduit](https://github.com/conduitio/conduit). If you want to implement a connector plugin in another language please
+have a look at the [connector plugin protocol](https://github.com/conduitio/connector-plugin).
+
+## Quickstart
+
+Create a new folder and initialize a fresh go module:
+```
+go mod init example.com/conduit-plugin-demo
+```
+
+Add the connector plugin SDK dependency:
+
+```
+go get github.com/conduitio/connector-plugin-sdk
+```
+
+You need to create two structs, one that implements `sdk.Source` and another that implements `sdk.Destination`. Apart
```

Comment:
one of these could be nil, maybe we should talk more about how one of these could be optional and how to only create a source or a destination connector

Reply:
Done, wdyt?

Reply:
looks good 👍